sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -289,53 +289,56 @@ case class Elt(children: Seq[Expression])
     val index = indexExpr.genCode(ctx)
     val strings = stringExprs.map(_.genCode(ctx))
     val indexVal = ctx.freshName("index")
+    val resultState = ctx.freshName("eltResultState")
```

Comment:
maybe this can have a better name now that it is a boolean.... I am not very good at naming, but something like `indexFound` or anything you feel appropriate...

Reply:
Ok.